COMMUNITY-ROLES.md

```diff
@@ -32,9 +32,9 @@ exceptions can always be considered, via open community discussion.)
   This means they will be able to push commits to that repository,
   as well as merge PRs, label and close issues, among other things.
 
-- **Collaborators who perform maintenance tasks should be made org members.**
-  (Maintenance work is understood as facilitating contributions by other people,
-  which in this project consists primarily of reviewing and/or merging PRs.)
+- **Repo collaborators who show interest in maintenance work should be added as org members.**
```

Comment:
Isn't "showing interest" more vague ? I would think performing maintenance tasks is clear and succinct.

Reply:
I see. My thinking was that "perform" sounds more like a one-off thing, while "show interest" implies a sustained activity. Perhaps we could instead change it to "who _regularly_ perform maintenance tasks"?

Reply:
Yeah, we can do that.

Reply:
I agree, that sounds way better.